db/db_basic_test.cc

```diff
@@ -1788,7 +1795,16 @@ TEST_P(DBBasicTestWithParallelIO, MultiGet) {
   ASSERT_TRUE(CheckValue(1, values[0].ToString()));
   ASSERT_TRUE(CheckValue(51, values[1].ToString()));
 
-  int expected_reads = random_reads + (fill_cache() ? 0 : 2);
+  bool read_from_cache = false;
+  if (fill_cache()) {
```

Comment:
This seems a bit confusing to me: why do we expect the code to *read* from cache only if `fill_cache` is enabled? For instance, the following scenario is perfectly possible: `fill_cache` is `false`, the block is present in the compressed block cache but not present in the uncompressed block cache. In this case, the code would retrieve the compressed block (from cache), uncompress it and return it to the user without inserting into the uncompressed cache.

Reply:
The test relies on ```fill_cache``` to populate the block cache in the first place. There will have to be some other way to warm up the cache. I'm not sure if its worth the extra complexity though, since it won't increase the coverage.

Reply:
Oh, I missed that `expected_reads` is the number of reads from the file. Makes sense now. Thanks!